KartingCoach/View models/CircuitDetailViewModel.swift

```diff
@@ -24,6 +28,9 @@ final class CircuitDetailViewModel: CircuitDetailViewModeling {
     
     var newRaceVM: NewRaceViewModeling { return factories.newRaceVMFactory(circuit.value) }
     
+    private let (reloadSignal, reloadObserver) = Signal<Void, NoError>.pipe()
+    lazy var reloadData: Signal<Void, NoError> = self.reloadSignal
```

Comment:
`self.races.signal.map { _ in }` by neslo?

Reply:
👍 vůbec mě to nenapadlo